internal/pkg/manifest/env.go

```diff
@@ -336,11 +336,20 @@ func (cfg publicHTTPConfig) IsEmpty() bool {
 }
 
 type privateHTTPConfig struct {
-	InternalALBSubnets []string `yaml:"subnets,omitempty"`
-	Certificates       []string `yaml:"certificates,omitempty"`
+	InternalALBSubnets   []string             `yaml:"subnets,omitempty"`
+	Certificates         []string             `yaml:"certificates,omitempty"`
+	SecurityGroupsConfig securityGroupsConfig `yaml:"security_groups,omitempty"`
```

Comment:
Just wanted to bring to your notice that my PR to add security_group to env manifest also have similar [secuirty_group](https://github.com/aws/copilot-cli/pull/3749/files#diff-4c63a9ce04347fa71ffd19988ac84f3eca7aacf36c414bb9e8f95c011b0f622dR101) field in the `vpc`. 
I was just concerned about having the same naming with just extra `s` at the end of `security_groups` will be little confusing for the customers.  What do you think about this?

Reply:
Yes you are correct. I was just making sure that 2 almost similar parameters shouldn't be confusing to the customers. But as you said they read differently in different sections should be okay.

Reply:
This is related to https://github.com/aws/copilot-cli/pull/3737/files#diff-4c63a9ce04347fa71ffd19988ac84f3eca7aacf36c414bb9e8f95c011b0f622dR327 . it is just `from_cdn` won't be valid for `private`.

```yaml
http:
  public:
    security_groups:
      ingress:
        from_cdn: true
```